docs/product-installation-guide/src/main/asciidoc/eap-business-central-install-proc.adoc

```diff
@@ -10,16 +10,14 @@ Business Central is a web console that enables you to perform the following task
 .Prerequisites
 * {EAP} 7.0.7 installed on your hard drive. The base directory of the EAP installation is referred to as `__EAP_HOME__`.
 * The following file, available from the site that contains this document:
-** Business Central:
-ifdef::BPMS[`jboss-bpmsuite-7.0.0.Beta02-business-central-eap7.zip`]
-ifdef::BRMS[`jboss-brms-7.0.0.Beta02-business-central-eap7.zip`]
++
+ifdef::BPMS[`jboss-bpmsuite-7.0.0.LA.ER3-collection.zip`]
+ifdef::BRMS[`jboss-brms-7.0.0.LA.ER3-collection.zip`]
 
 .Procedure
-. Extract the
-ifdef::BPMS[`jboss-bpmsuite-7.0.0.Beta02-business-central-eap7.zip`]
-ifdef::BRMS[`jboss-brms-7.0.0.Beta02-business-central-eap7.zip`]
-archive to an arbitrary directory, here referred to as `__TEMP_DIR__`.
-. Copy the contents of the `__TEMP_DIR__/jboss-eap-7.0/` directory into `__EAP_HOME__`. When asked to overwrite files or merge directories, select *Yes*.
+. Extract the `jboss-bpmsuite-7.0.0.LA.ER3-collection.zip` archive to a temporary directory. In the following examples this directory is called `__TEMP_DIR__`.
```

Comment:
Shouldn't the .zip file name also be ifdef dependent? BRMS vs BPM Suite, as in the prereqs?